packages/react-client/src/hooks/useFishjamClient.ts

```diff
@@ -94,7 +94,11 @@ function endpointToPeerState(
   };
 }
 
-export const useFishjamClient = () => {
+/*
+This is an internally used hook.
+It is not meant to be used by the end user.
+*/
+export const useINTERNAL_FishjamClient = () => {
```

Comment:
Maaaaybe:
```suggestion
export const useFishjamClient_DONOTUSE = () => {
```

Reply:
only thing i dont like about it is that the words blend in uppercase

Reply:
I don't have any preferences, but if we use `_` in the middle, we could also write is as `useFishjamClient_DO_NOT_USE`

Reply:
`useFishjamClient_DO_NOT_USE` is also fine, let's change it and then it is ready to merge :)